data.sql

```diff
@@ -4,3 +4,11 @@ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Gabumon', '2018-11-15', 2, 'TRUE', 8);
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Pikachu', '2021-01-07', 1, 'FALSE', 15.04);
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Devimon', '2017-05-12', 5, 'TRUE', 11);
+
+/* Alter Tables Animal */
+ALTER TABLE animals ADD COLUMN species varchar;
```

Comment:
- [ ] Kindly this query change the structure of the table, it would be better to be added in the `schema.sql file` not in the `data` as suggested in the project requirements 👍 

![image](https://user-images.githubusercontent.com/102757126/229293712-2c0eeb9c-d4f5-41d1-8b6b-b57b50b6faaa.png)